.github/workflows/buildcli.yml

```diff
@@ -0,0 +1,148 @@
+# This pipeline is used as template to build the sfpowerscripts plugin
+
+
+name: 'sfpowerscripts Plugin Build'
+
+on:
+  workflow_call:
+    inputs:
+      commitToGit:
+        type: boolean
+        default: false
+      stageToDeploy:
+        type: string
+        default: review
+      version:
+        type: string
+        default: ''
+      environmentToDeploy:
+        type: string
+        default: development
+      organization:
+        type: string
+        default: ''
+      buildNumber:
+        type: string
+        default: ''
+      toBuild:
+        type: boolean
+        default: false
+      bump:
+        type: string
+        default: 'patch'
+      publish:
+        type: boolean
+        default: false
+
+
+jobs:
+  build:
+    name: 'Build Plugin'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: 'Update Build Number'
+        if: ${{ inputs.version == 'alpha' }} || ${{ inputs.version == 'hotfix' }}
+        run: |
+          PKG_VERSION=$(jq -r ".version" package.json)
+          echo "##vso[build.updatebuildnumber]${PKG_VERSION}"
+        working-directory: '$(System.DefaultWorkingDirectory)/packages/sfpowerscripts-cli'
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14' 
+
+      - name: 'Set Git Config'
+        run: |
+          git config --global user.email "sfpowerkit.ci@dev.azure.com"
+          git config --global user.name "sfpowerkitci"
+
+
+      - name: 'Create .npmrc file'
+        run: |
+           echo "${{ env.NPM_SCOPE }}:registry=https://registry.npmjs.org" > ~/.npmrc
+           echo  "_authToken=${AUTH_TOKEN}" >> ~/.npmrc
+        env:
+          AUTH_TOKEN: ${{ secrets.NPM_TOKEN }} #Need to change this
+
+      - name: 'Create packages/core/.npmrc file'
+        run: |
+           echo "${{ env.NPM_SCOPE }}:registry=https://registry.npmjs.org" > packages/core/.npmrc
+           echo  "_authToken=${AUTH_TOKEN}" >> packages/core/.npmrc
+        env:
+          AUTH_TOKEN: ${{ secrets.PKG_CORE_NPM_TOKEN }} #Need to change this
+
+      - name: 'Create packages/sfpowerscripts-cli/.npmrc file'
+        run: |
+           echo "${{ env.NPM_SCOPE }}:registry=https://registry.npmjs.org" > packages/sfpowerscripts-cli/.npmrc
+           echo  "_authToken=${AUTH_TOKEN}" >> packages/sfpowerscripts-cli/.npmrc
+        env:
+          AUTH_TOKEN: ${{ secrets.PKG_SFPCLI_AUTH_TOKEN }} #Need to change this
+
+# TODO: find equivalent for gh actions
```

Comment:
I don't think there is an equivalent action for npm authentication. 
Authentication can be achieved by copying the `.npmrc` file into the HOME directory instead of the package directories above.